### PR TITLE
Advanced usage docs

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,42 +117,111 @@ dependencies {
...
-        String beaconUrl = "<realm>";
-        String rumAccessToken = "<your_RUM_access_token>";
-        Config config = SplunkRum.newConfigBuilder()
-                .realm(realm)
-                .rumAccessToken(rumAuth)
-                .applicationName("My Android App")
-                .build();
+  String beaconUrl="<realm>";
+  String rumAccessToken="<your_RUM_access_token>";
+  Config config=SplunkRum.newConfigBuilder()
```
Comment (mateuszrzeszutek) [inline README.md:134]: Nit: spaces around `=` looked a bit nicer

**Review by mateuszrzeszutek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jkwatson) [reply README.md:134]: yeah, Android Studio keeps trying to format the markdown and messing it up. 😢

Comment (jkwatson) [reply README.md:134]: I think I'm going to put a full java class in here to see if it does a better job not messing it up.

Comment (jkwatson) [reply README.md]: done, and IDEA is much happier formatting it now

**Pull request merged**